app/lib/smaxtec_api.rb

```diff
@@ -31,6 +56,11 @@ def update_sensor_readings
 
 
   def smaxtec_sensor_readings(sensor)
```

Comment:
I consider the mixture of `event` and `sensor_reading` code smell.

Create two separate methods for events and sensor readings, please.